scipy/optimize/_linprog_ip.py

```diff
@@ -228,15 +237,15 @@ def _get_delta(A, b, c, x, y, z, tau, kappa, gamma, eta, sparse=False,
             if ip:  # if the correction is to get "initial point"
                 # Reference [4] Eq. 8.23
                 rhatxs = ((1 - alpha) * gamma * mu -
-                          x * z - alpha**2 * d_x * d_z)
+                          x * z - alpha ** 2 * d_x * d_z)
```

Comment:
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)"
See [here in PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations).
```suggestion
                          x * z - alpha**2 * d_x * d_z)
```

Generally, please do not make style changes unrelated to a PR. Currently the files pass PEP8 style checks, so changes will usually be subjective, and review time is better spent on enhancements and bug fixes.

Thank you!

Reply:
Thanks for the review. All style changes were unintentional and done by the IDE I was using.